package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puzzle-lib",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
I usually save this for when I publish. I think technically this is semver minor which means this will publish as 1.5.0.

Reply:
Reverted